BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -340,8 +385,25 @@ public final class ConvivaAnalytics: NSObject {
         buildDynamicContentMetadata()
     }
 
+    private func buildAdMetadata() -> [String: Any] {
+        let includedTags: Set<String> = [
+            CIS_SSDK_METADATA_STREAM_URL,
+            CIS_SSDK_METADATA_IS_LIVE,
+            CIS_SSDK_METADATA_DEFAULT_RESOURCE,
+            CIS_SSDK_METADATA_ENCODED_FRAMERATE,
+            "streamType",
+            "integrationVersion",
+            CIS_SSDK_METADATA_VIEWER_ID,
+            CIS_SSDK_METADATA_PLAYER_NAME,
```

Comment:
These two are listed in the Conviva documentation as "autocollected" from the main video session but during testing they were showing up as error due to missing values in the ad session, therefore we decided to explicitly set these for the ad info.